### DISCUSSION TITLE
Working example of gym_anytrading

### DISCUSSION THREAD

**Discussion by yourGuy** [opendilab/DI-engine#756] [Q&A]
Does anybody have a working example of using the gym_anytrading environment. I'm struggling with initiating the model .
Thank you

**Comment by Cloud-Pku** [comment]
Hello, this is a [demo (colab)](https://colab.research.google.com/drive/1IKfjfyb0PSyTranJemvve2V38_JUs6yB?usp=sharing) for gym-anytrading.
